CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.10.1] - 2023-01-18
```

Comment:
I don't think this has the comparison link.

Reply:
fixed